### PR TITLE
Add QTLTools package.

### PR DESCRIPTION
**Pull request by dunatotatos**

### CODE REVIEW

File: var/spack/repos/builtin/packages/qtltools/package.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class Qtltools(MakefilePackage):
+    """A complete tool set for molecular QTL discovery and analysis."""
+
+    # FIXME: Add a proper url for your package's homepage here.
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/qtltools/package.py]: Can you remove this?

File: var/spack/repos/builtin/packages/qtltools/package.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    homepage = "https://qtltools.github.io/qtltools/"
+    url      = "https://github.com/qtltools/qtltools/archive/refs/tags/1.3.1.tar.gz"
+
+    version('1.3.1', sha256='033b9b61923fd65c4b8b80bc0add321e6fd6fb40de49d15c2dfe6a4d7e60764a')
+    version('1.3', sha256='032020d7e038eac4ec01701343a887bced7cca356cbd24b3d5bbadf83686faeb')
+
+    depends_on('boost +pic +iostreams +program_options')
+    depends_on('gsl')
+    depends_on('htslib -libcurl -libdeflate')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/qtltools/package.py]:
```suggestion
    depends_on('htslib ~libcurl ~libdeflate')
```

**Review by alalazo — CHANGES_REQUESTED** [review]
Besides the two minor comments, style needs to be fixed too

File: var/spack/repos/builtin/packages/qtltools/package.py (hunk 1 above)
Comment (dunatotatos) [reply var/spack/repos/builtin/packages/qtltools/package.py]: I missed this one. Done.

**Review by alalazo — APPROVED** [review]

**Pull request merged**